reference/runtime-environment.html.md

```diff
@@ -32,6 +32,21 @@ Not to be confused with the [HTTP header](/docs/reference/runtime-environment/#f
 ### `FLY_IMAGE_REF`
 **Docker Image Reference**: The name of the Docker image running this container. Useful if your application needs to launch Machine instances of itself to scale up-or-down. Read about how [a Rails app runs Machines for scale-to-zero background workers](https://fly.io/ruby-dispatch/rails-background-jobs-with-fly-machines/). `registry.fly.io/my-app-name:deployment-01H9RK9EYO9PGNBYAKGXSHV0PH` is an example of the Docker Image Reference's format.
 
+### `FLY_MACHINE_ID`
+**Machine ID**: Each Fly Machine is identified by a unique Machine ID. This is the ID that `flyctl` and the `Machines API` use to identify that Machine, and you'll see it in the [logs](/docs/flyctl/logs/).
+
+### `FLY_MACHINE_VERSION`
+**Machine Configuration Version**: Every Machine configuration has an associated version. Whenever you update a Machine's configuration (including when you update its Docker image), it gets a new `FLY_MACHINE_VERSION`. The exception to this is that changing only the Machine's metadata doesn't trigger a new version.
```

Comment:
```suggestion
**Machine Configuration Version**: The version associated to a specific Machine configuration. When you update a Machine's configuration (including when you update its Docker image), it gets a new `FLY_MACHINE_VERSION`. Changing the Machine's metadata doesn't trigger a new version.
```

Suggesting some simplification, but feel free to change. 

My big question is, where do users see this? Is is the "fly_release_version"? Or is the "Instance ID" that you see when you run `fly m status`? Instance ID seems to correspond to Machine config version in admin.

Reply:
Sorry I'm having a hard time understanding the question. Could you possibly rephrase it for me?

Reply:
Good catch! @Lucais11 had in the draft that this is the Instance ID that you see in the output of `fly machine status`. It's my fault it got lost.

Reply:
Oh okay I get what was being asked now. Yes it is in the output of `fly machine status`.